packages/data/addon/serializers/elide-metadata.ts

```diff
@@ -79,9 +82,9 @@ export default class ElideMetadataSerializer extends EmberObject {
       const newTableDimensions = this._normalizeTableDimensions(table.dimensions, table.id, source);
       const newTableTimeDimensions = this._normalizeTableTimeDimensions(table.timeDimensions, table.id, source);
 
-      newTable.metricIds = newTableMetrics.map((m: MetricMetadata) => m.id);
-      newTable.dimensionIds = newTableDimensions.map((d: DimensionMetadata) => d.id);
-      newTable.timeDimensionIds = newTableTimeDimensions.map((d: TimeDimensionMetadata) => d.id);
+      newTable.metricIds = newTableMetrics.map((m: MetricMetadataPayload) => m.id);
```

Comment:
Can we make `_normalizeTableMetrics`/etc specify it's return type so we don't have to add the type here